python/sglang/srt/hardware_backend/npu/graph_runner/eagle_draft_extend_npu_graph_runner.py

```diff
@@ -37,6 +37,9 @@ def __init__(self, eagle_worker: EAGLEWorker):
     def _create_graph(self):
         return torch.npu.NPUGraph()
 
+    def _cache_loc_dtype(self):
+        return torch.int32
+
```

Comment:
```suggestion
    def get_cache_loc_dtype(self):
        return torch.int32

```
make this a public function that can be called outside